dhis-2/dhis-web-api-test/pom.xml

```diff
@@ -15,6 +15,39 @@
 
   <build>
     <finalName>dhis-web-api-test</finalName>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <configuration>
+          <failOnWarning>true</failOnWarning>
+          <ignoredUnusedDeclaredDependencies>
+          </ignoredUnusedDeclaredDependencies>
```

Comment:
Did you declared the ignored dependencies here to not mess with the main pom as this can be temporary?

Reply:
oh, I might have left this one on accident. Removed it, lets see if everything is still fine.